email-worker/pnpm-lock.yaml

```diff
@@ -0,0 +1,1764 @@
+lockfileVersion: '9.0'
```

Comment:
I dont think this is correct, we should be using pnpm workspaces to shasre deps between the website and the worker.It shouldnt generate a 2nd lock file i think.